src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -16,6 +16,13 @@ kernel = "@KERNELPATH@"
 image = "@IMAGEPATH@"
 machine_type = "@MACHINETYPE@"
 
+# Enable guest protection.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one.
```

Comment:
I think that the meaning for `true` to enable all we can makes sense.
But @sameo pointed out that there may be cases where we may want some subset of features, and I believe this may apply here. So may be instead of `true` and `false`, have `none` and `all` as a starting point, leaving room in the future if we need finer-grained control.

Reply:
ehmm.. the proposal was updated https://github.com/kata-containers/kata-containers/issues/1332#issuecomment-810462430

> I guess we could live with a boolean for now and detect the platform setting.